### ISSUE TITLE
Expose FocusTrap API through CdkTrapFocus

### ISSUE THREAD

**Issue by jelbourn** [angular/components#7979]
The `CdkTrapFocus` directive exposes its `FocusTrap` instance, requiring users to go through something like `this._focusTrap.focTrap.doSomething()`. 

We should make the internal `FocusTrap` private and expose the same APIs

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
